test/unit/cmap/connection.test.ts

```diff
@@ -137,6 +157,89 @@ describe('new Connection()', function () {
     });
   });
 
+  describe('#onMessage', function () {
+    context('when the connection is a monitoring connection', function () {
+      let queue: Map<number, OperationDescription>;
+      let driverSocket: FakeSocket;
+      let connection: Connection;
+
+      beforeEach(function () {
+        driverSocket = sinon.spy(new FakeSocket());
+        // @ts-expect-error: driverSocket does not fully satisfy the stream type, but that's okay
+        connection = sinon.spy(new Connection(driverSocket, connectionOptionsDefaults));
+        connection.isMonitoringConnection = true;
+        const queueSymbol = getSymbolFrom(connection, 'queue');
+        queue = connection[queueSymbol];
+      });
+
+      context('when requestId/responseTo do not match', function () {
+        let callbackSpy;
+        const document = { ok: 1 };
+
+        beforeEach(function () {
+          callbackSpy = sinon.spy();
+          // Create the operation description.
+          const operationDescription: OperationDescription = {
+            requestId: 1,
+            cb: callbackSpy
+          };
+
+          // Stick an operation description in the queue.
+          queue.set(1, operationDescription);
+          // Emit a message that won't match the existing operation description.
+          const msg = generateOpMsgBuffer(document);
+          const msgHeader: MessageHeader = {
+            length: msg.readInt32LE(0),
+            requestId: msg.readInt32LE(4),
+            responseTo: msg.readInt32LE(8),
```

Comment:
This will default to 0 so won't match.

Reply:
can we add this as a code comment instead so that if we need to modify these tests later it's more clear what the line is doing?